exercises/minesweeper/minesweeper.ml

```diff
@@ -0,0 +1,53 @@
+open Base
```

Comment:
It looks like this is an example solution, but it is placed in the stub file.

Having separate pull requests per exercise that is updated is preferable to shorten the review.

Reply:
Thanks for catching this :)